_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -747,6 +747,10 @@ public function update_data( $request ) {
 				case 'pinterest':
 				case 'yandex':
 					$grouped_options          = $grouped_options_current = (array) get_option( 'verification_services_codes' );
+					$matches = array();
+					if ( preg_match( '#^<meta name="([a-z0-9_\-.:]+)?" content="([a-z0-9_-]+)?" />$#i', $value, $matches ) ) {
+						$value = $matches[ 2 ];
+					}
```

Comment:
I think we could simplify the regular expression with:

```regex
#.*<meta name="(?:[^"]+)" content="([^"]+)" />.*#i
```

This makes use of a [non-capturing subpattern](https://secure.php.net/manual/en/regexp.reference.subpatterns.php) since we don't care about the `name` attribute (we should get `content`  from `$matches[1]` now though). It also uses a [character class](https://secure.php.net/manual/en/regexp.reference.character-classes.php) that should achieve the same result, while simpler. Finally, it is now more lenient with potential heading and trailing white spaces.